.github/helm/affine-cloud/templates/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: affine-cloud
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: affine-cloud
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 2
+  template:
+    metadata:
+      labels:
+        app: affine-cloud
+    spec:
+      restartPolicy: Always
+      containers:
+      - name: affine-cloud
+        image: "ghcr.io/toeverything/cloud-self-hosted:{{ .Values.affineCloud.tag | default .Chart.AppVersion }}"
+        env:
+        - name: DATABASE_URL
+          value: "postgresql://$(POSTGRES_USER):$(POSTGRES_PASSWORD)@postgres-service/affine"
```

Comment:
I'd like to make the whole `DATABASE_URL` configurable by the helm `values`; and make `postgresql://$(POSTGRES_USER):$(POSTGRES_PASSWORD)@postgres-service/affine` as the default value